blog/2017-03-15/index.php

```diff
@@ -26,8 +26,8 @@
 require_once("../../header.php");
 ?>
 
-<link rel="stylesheet" href="<?php echo $site_root; ?>media/fancybox/jquery.fancybox-1.3.4.css" media="screen">
-<script src="<?php echo $site_root; ?>jquery.min.js"></script>
```

Comment:
I would love to see documentation on this magic CDN switch, I see nothing about it anywhere when searching.